GrandCentralBoard/Widgets/Harvest/HarvestStats.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  HarvestStats.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+
+struct HarvestTeamStats {
+    let updateDate: NSDate
+    let dailyStats: [HarvestDailyTeamStats]
+}
+
+struct HarvestDailyTeamStats {
+    let day: NSDate
+    let groups: [HarvestTeamStatsGroupType: HarvestTeamStatsGroup]
+}
+
+struct HarvestTeamStatsGroup {
+    let type: HarvestTeamStatsGroupType
+    let count: Int
+    let averageWorkTime: NSTimeInterval
+}
+
+enum HarvestTeamStatsGroupType {
```

Comment:
It seems to long and Harvest is not needed here. 

`TimeRange` ?

Also you can add  `let count: Int` and name it:

`TimeRangeStatistics`